src/main/java/org/benetech/servicenet/adapter/sheltertech/mapper/ShelterTechLocationMapper.java

```diff
@@ -5,33 +5,39 @@
 import org.benetech.servicenet.domain.PhysicalAddress;
 import org.benetech.servicenet.domain.PostalAddress;
 import org.mapstruct.Mapper;
-import org.mapstruct.Mapping;
 import org.mapstruct.Named;
 import org.mapstruct.factory.Mappers;
 
-import static org.benetech.servicenet.adapter.sheltertech.ShelterTechConstants.PROVIDER_NAME;
+import java.util.Collections;
+import java.util.Set;
 
 @Mapper
 public interface ShelterTechLocationMapper {
 
     ShelterTechLocationMapper INSTANCE = Mappers.getMapper(ShelterTechLocationMapper.class);
 
-    @Mapping(ignore = true, target = "id")
-    @Mapping(source = "address1", target = "name", defaultValue = "undefined")
-    @Mapping(ignore = true, target = "alternateName")
-    @Mapping(ignore = true, target = "description")
-    @Mapping(ignore = true, target = "transportation")
-    @Mapping(source = "raw.latitude", target = "latitude")
-    @Mapping(source = "raw.longitude", target = "longitude")
-    @Mapping(source = "raw.id", target = "externalDbId")
-    @Mapping(constant = PROVIDER_NAME, target = "providerName")
-    @Mapping(ignore = true, target = "physicalAddress")
-    @Mapping(ignore = true, target = "postalAddress")
-    @Mapping(ignore = true, target = "regularSchedule")
-    @Mapping(ignore = true, target = "holidaySchedule")
-    @Mapping(ignore = true, target = "langs")
-    @Mapping(ignore = true, target = "accessibilities")
-    Location mapToLocation(AddressRaw raw);
+    default Set<Location> mapToLocation(AddressRaw raw) {
```

Comment:
You could keep the old method
```suggestion
Location mapToLocation(AddressRaw raw);
```
 and simply add 
```suggestion
Set<Location> mapToLocations(List<AddressRaw> raws);
```
see https://github.com/benetech/ServiceNet/blob/39c4eb7f729c8c466506fe1266a7399bf1e0ba4f/src/main/java/org/benetech/servicenet/adapter/sheltertech/mapper/ShelterTechPhoneMapper.java#L27